src/Compilers/Core/Portable/Operations/ITranslatedQueryExpression.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents an unrolled/lowered query expression in C# and VB.
+    /// For example, for the query expression "from x in set where x.Name != null select x.Name", the select clause is the last clause of the unrolled query expression,
```

Comment:
>clause [](start = 109, length = 6)

It feels like the comment should be adjusted to no longer refer to query clauses as we are not exposing any special nodes to represent them.

Reply:
Sure, I will do so.

---
In reply to: [137398396](https://github.com/dotnet/roslyn/pull/21356#discussion_r137398396) [](ancestors = 137398396)